### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by uplight-dev** [lachlan2k/wag-docker#1]
```
#0 6.027 In file included from /app/wag/internal/router/xdp.c:3:
#0 6.027 /usr/include/linux/types.h:5:10: fatal error: 'asm/types.h' file not found
#0 6.027 #include <asm/types.h>
#0 6.027          ^~~~~~~~~~~~~

```

**Comment by jasonwee** [comment]
yeap, me too

```
go: downloading github.com/go-webauthn/revoke v0.1.9
go: downloading github.com/x448/float16 v0.8.4
In file included from /app/wag/internal/router/xdp.c:3:
/usr/include/linux/types.h:5:10: fatal error: 'asm/types.h' file not found
#include <asm/types.h>
         ^~~~~~~~~~~~~
1 error generated.
Error: can't execute clang: exit status 1
exit status 1
internal/router/bpf.go:29: running "go": exit status 1
make: *** [Makefile:18: .generate_ebpf] Error 1
The command '/bin/sh -c make' returned a non-zero code: 2
```

**Comment by uplight-dev** [comment]
@jasonwee added a new PR for fixing this, feel free to take it in if cannot wait PR merge

**Comment by lachlan2k** [comment]
Closing as addressed by #2 , thanks again
